### ISSUE TITLE
Strange address from client.members()

### ISSUE THREAD

**Issue by charleswhchan** [KushalP/serfclient-py#20]
I am using serf 0.6.4 and serfclient 0.5.0. Following the instructions, I tried to use client.members(), but the address format is some what cryptic. How should I interpret the 'Addr' field?

If I use `serf members` directly from CLI, it shows the:
```
node-9         192.168.xx.yyy:7946  alive
...
```

Whereas if I use client.members():
```
{  
   'Members':[  
      {  
         'Status':'alive',
         'DelegateCur':4,
         'Addr':'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\xff\xff\xc4\xa8?\x91',
         'Tags':{  

         },
         'ProtocolMax':2,
         'DelegateMin':2,
         'ProtocolMin':1,
         'ProtocolCur':2,
         'Port':7946,
         'DelegateMax':4,
         'Name':'node-9'
      },
      {  
         'Status':'alive',
         'DelegateCur':4,
         'Addr':'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\xff\xff\n\x05\x02\x0f',
         'Tags':{  

         },
         'ProtocolMax':2,
         'DelegateMin':2,
         'ProtocolMin':1,
         'ProtocolCur':2,
         'Port':7946,
         'DelegateMax':4,
         'Name':'node-1'
      },
      {  
         'Status':'alive',
         'DelegateCur':4,
         'Addr':'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\xff\xff\xc0\xb8?\x86',
         'Tags':{  

         },
         'ProtocolMax':2,
         'DelegateMin':2,
         'ProtocolMin':1,
         'ProtocolCur':2,
         'Port':7946,
         'DelegateMax':4,
         'Name':'node-3'
      },
      {  
         'Status':'alive',
         'DelegateCur':4,
         'Addr':'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\xff\xff\xc1\xa8?{',
         'Tags':{  

         },
         'ProtocolMax':2,
         'DelegateMin':2,
         'ProtocolMin':1,
         'ProtocolCur':2,
         'Port':7946,
         'DelegateMax':4,
         'Name':'node-5'
      }
   ]
}
```

The expected output format as defined by Hashicorp:
https://www.serfdom.io/docs/agent/rpc.html
```
    {"Members": [
        {
        "Name": "TestNode"
        "Addr": [127, 0, 0, 1],
        "Port": 5000,
        "Tags": {
            "role": "test"
        },
        "Status": "alive",
        "ProtocolMin": 0,
        "ProtocolMax": 3,
        "ProtocolCur": 2,
        "DelegateMin": 0,
        "DelegateMax": 1,
        "DelegateCur": 1,
        },
        ...]
    }
```

It appears the array elements are not unpacked correctly.

**Comment by charleswhchan** [comment]
@KushalP, any chance you might be able to take a look at this issue?

**Comment by KushalP** [comment]
@charleswhchan Hi Charles, I've not had any time free to prioritise this yet. If you'd like to, feel free to submit a PR for it. I think the solution will involve writing a custom unpacker for the field: https://pypi.python.org/pypi/msgpack-python/#packing-unpacking-of-custom-data-type

**Comment by charleswhchan** [comment]
Thanks @KushalP, I will take a look into it.

**Comment by charleswhchan** [comment]
Related to https://github.com/hashicorp/serf/issues/158

**Comment by charleswhchan** [comment]
PR: https://github.com/KushalP/serfclient-py/pull/21

**Comment by KushalP** [comment]
This is now fixed as part of #21
